### ISSUE TITLE
Drop target font color is wrong

### ISSUE THREAD

**Issue by 7even11** [JAM-Software/Virtual-TreeView#1083]
When doing drag and drop from one VirtualStringTree to another, while hovering over an item, it uses the color **Colors.DropTargetBorderColor** and **Colors.DropTargetColor** for background (all good) but for for the text, it continues to use **Colors.UnfocusedColor**.

So instead of having inverted color text (usually white) on blue background by default, it displays black text on blue background which doesn't look good and with somewhat darker **clHighlight** it can be entirely unreadable. Previous versions (4.7.*) were using correct colors, inverted text on darker background.

Possible solution would be to introduce another color for drag and drop for example **Colors.DropTargetTextColor** with the default set to for example **clHighlightText** (which is white usually) or to invert the text color properly.

![Wrong DragDrop color](https://i.imgur.com/eS7leIQ.png)

Also, while dragging in the minimal example, there is a lot of errors like:

Debug output:
onecore\com\combase\dcomrem\resolver.cxx(2507)\combase.dll!7790CF4D: (caller: 778929E4) ReturnHr(3652) tid(1534) 80040154 Class not registered

Which might be another bug perhaps.

**Comment by joachimmarder** [comment]
I did a quick test with our TreeSize application and the Advanced sample project, and all looked as expected. There are lot of settings that influence the colors of the Virtual TreeView control.

Please respect our guidelines on the project homepage for submitting bugs. Please include the version of Virtual TreeView and Delphi, and a sample projects or instructions how to modify one of the included sample projects to reproduce the problem.

> Previous versions (4.7.*) ...

It would be helpful if you can narrow down the version number in which the behavior has changed.

> Also, while dragging in the minimal example, there is a lot of errors like:
> Debug output:
> onecore\com\combase\dcomrem\resolver.cxx(2507)\combase.dll...

These are not errors, but debug output issued through the Windows API function `OutputDebugString()`. In this case, it seems the Windows DLL `Combase.dll` issued this debug output.

**Comment by 7even11** [comment]
Unfortunately, I jumped from 5.5.3 directly to 7.61 and I didn't see which exact version changed the behavior. It was the 5.5.3 where this last worked properly to my knowledge. It should be fixable by introducing a color for text when it is dragged over a node. Attached with this is an example project file, for Delphi 10.4.2, where I used VirtualTreeView 7.61, running on latest update of Windows 10 21H2.

To avoid reporting with heavily tweaked component settings, I started off with default settings on each VirtualTreeView (minimal modifications just to add drag and drop repro example).

[VSTBugs.zip](https://github.com/JAM-Software/Virtual-TreeView/files/8011380/VSTBugs.zip)
